components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -93,6 +93,15 @@ std::string BuildReferralEndpoint(const std::string& path) {
 
 namespace brave {
 
+std::string GetAPIKey() {
+  std::string api_key = BRAVE_REFERRALS_API_KEY;
```

Comment:
This should also not be using an env var, but why are we using referrals api key as an api key for webcompat?

Reply:
referrals API key was chosen as a way to raise the level of effort needed for someone to spam the endpoint without introducing complexity, cc @aekeus